README.md

```diff
@@ -62,6 +69,21 @@ Format all project files as follows:
 sbt scalafmtAll
 ```
 
+## Custom commands
+
+### Pre-Commit
+
+This is a sbt command alias specific to this project. It will clean, compile then run a scala format of the
+code and build.sbt files.
+
+> `sbt preCommit`
+
+### Format all
+
+This is a sbt command alias specific to this project. It will format of the code and build.sbt files.
```

Comment:
Little typo here with of the code

`This is a sbt command alias specific to this project. It will format the code and build.sbt files.`

Reply:
Good spot. Done